packages/vuepress-theme-odyssey/components/DocsPageHeader.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <header
+    :class="{
+      'docs-page-header': true,
+      [`is-page-header-${variant}`]: true
+    }"
+  >
+    <h1 class="docs-page-header--title">{{ title }}</h1>
+    <p v-if="lead" class="docs-page-header--lead">{{ lead }}</p>
+    <slot name="right"></slot>
```

Comment:
> We shouldn't use directional names - prep for internationalization
> 
> _Originally posted by @edburyenegren-okta in https://github.com/okta/odyssey/pull/714#discussion_r509604002_

Good feedback! Any ideas on what to call this?

Reply:
Sorry, totally missed this after the transfer. Maybe... Meta? External? Links?